cmd/vulcan-nuclei/main.go

```diff
@@ -167,7 +161,7 @@ func processNucleiFindings(target string, nucleiFindings []ResultEvent) []*repor
 			}
 		}
 		var vuln = report.Vulnerability{
-			AffectedResource: target,
+			AffectedResource: v.Matched,
```

Comment:
I'd really like to see if there's any kind of variability of this field between executions.

Reply:
We can do some tests on this, but it should not vary at all unless there is an actual change in the target. For network checks (such as the ones we are currently discussing in #241, #242, #243) this string will just be the hostname plus the port. In the case of web addresses the full URL where the check found the issue. It could happened that, for web applications, the vulnerability can be found in different places and different ones are reported each time, similar to what happens with the ZAP and Burp checks where the crawling is not exhaustive nor deterministic. However, this still means that the vulnerability was found in different places.

Reply:
Cool, thx.